{
  "rustsec_label": "Patched",
  "in_rustsec_patched": true,
  "in_rustsec_current": false,
  "in_rust_sec": true,
  "rustsec_tag": "Vulnerability",
  "developers": [
    "fafhrd91",
    "github:pyo3:admins"
  ],
  "stars": 13288,
  "forks": 813,
  "downloads": 68211152,
  "num_side_effects": 1040,
  "failed_rudra": false,
  "audits": [],
  "dependencies": [
    [
      "cfg-if",
      "1.0.0"
    ],
    [
      "libc",
      "0.2.171"
    ],
    [
      "memoffset",
      "0.9.1"
    ],
    [
      "once_cell",
      "1.21.1"
    ],
    [
      "portable-atomic",
      "1.11.0"
    ],
    [
      "pyo3-ffi",
      "0.24.0"
    ]
  ],
  "passed_audit": false,
  "num_unsafe_calls": 672,
  "miri": true,
  "past_audit": false,
  "miri_details": {
    "status": "crash",
    "passed": 0,
    "failed": 3,
    "ignored": 0,
    "measured": 0,
    "filtered_out": 0,
    "time_seconds": 0.0
  }
}